### PR TITLE
docs(recipes): Active users recipe

### PR DESCRIPTION
**Pull request by rchkv**
Changed the Active users' recipe to a common format.
Preview: https://deploy-preview-3310--cubejs-docs-staging.netlify.app/recipes/active-users

### CODE REVIEW

File: docs/content/Examples-Tutorials-Recipes/Active-users.md [hunk 1]
```diff
@@ -8,88 +8,119 @@ redirect_from:
...
-trailing, leading, and offset. You can read about what each of them does
-[here](/schema/reference/measures#parameters-rolling-window).
-
-For our purpose, we need only offset and trailing. We will set offset to
-<b>start</b> and the trailing parameter to the number of days – 1, 7, or 30.
+Daily, weekly, and monthly active users are commonly referred to as DAU, WAU,
+MAU. To get these metrics, we need to use a rolling time frame to calculate a
+daily count of how many users interacted with the product or website in the
+prior day, 7 days, or 30 days. Also, we can build other metrics on top of these
+basic metrics. For example, the WAU to MAU ratio. We can add it, using already
```
Comment (rpaik) [inline docs/content/Examples-Tutorials-Recipes/Active-users.md:22]:
```suggestion
basic metrics. For example, the WAU to MAU ratio, which we can add by using already
```

File: docs/content/Examples-Tutorials-Recipes/Active-users.md [hunk 2]
```diff
@@ -8,88 +8,119 @@ redirect_from:
...
 });
 ```
 
-Going further, we can build other metrics on top of these basic metrics. For
-example, <b>the DAU to MAU ratio</b> is one of the most popular metrics used to
-<b>measure the stickiness of the product</b>. We can easily add it, using
-already defined `dailyActiveUsers` and `monthlyActiveUsers`.
+## Query
+
+We should set a `timeDimensions` with the `dateRange` in the query that will
```
Comment (rpaik) [inline docs/content/Examples-Tutorials-Recipes/Active-users.md:80]: Should this be "... in the query that will be sent to the Cube API"?

**Comment by rpaik** [comment]
Just a few minor suggestions 😊

File: docs/content/Examples-Tutorials-Recipes/Active-users.md (hunk 2 above)
Comment (rchkv) [reply docs/content/Examples-Tutorials-Recipes/Active-users.md:80]: Yeah, I guess we can delete that :)

**Pull request merged**